components/Day.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+  icon: {
+    height: 55,
+    width: 55,
+  },
+  floatRight: {
+    alignSelf: 'flex-end',
+  },
+  floatleft: {
+    alignSelf: 'flex-start',
+  },
+  baseText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 18,
+    textAlign: 'center',
+  },
+  smallText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 14,
+    textAlign: 'center',
+  },
+});
+
+const Day = (props) => {
+  const { weather, style } = props;
+  return (
+    <View style={style}>
+      <Text style={styles.baseText}>{weather.name}</Text>
+
+      <View
+        style={{
+          flexDirection: 'column',
+          flex: 1,
+          //   backgroundColor: 'rgba(0,0,100,0.4)',
+        }}>
+        <Image
+          style={styles.icon}
+          resizeMode="contain"
+          source={{
+            uri: `http://openweathermap.org/img/wn/${weather.iconDay}@2x.png`,
```

Comment:
Move the host part to the config and use url api for building it. https://developer.mozilla.org/en-US/docs/Web/API/URL/URL